Source: http://lxde.svn.sourceforge.net/viewvc/lxde/trunk/lxshortcut/configure.in?r1=1737&r2=1823&view=patch
Upstream: Yes
Reason: patch solved the problem on runs configure in infinite loop
--- lxshortcut/configure.in	2009/07/05 23:53:22	1737
+++ lxshortcut/configure.in	2009/07/15 12:40:26	1823
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
